scm-ui/ui-components/src/navigation/PrimaryNavigationLink.tsx

```diff
@@ -21,42 +21,32 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import * as React from "react";
-import { Route, Link } from "react-router-dom";
+import React, { FC } from "react";
+import { useHistory, useRouteMatch } from "react-router-dom";
 import { createAttributesForTesting } from "../devBuild";
+import classNames from "classnames";
 
 type Props = {
   to: string;
+  match: string;
   label: string;
-  match?: string;
-  activeOnlyWhenExact?: boolean;
   testId?: string;
+  className?: string;
 };
 
-class PrimaryNavigationLink extends React.Component<Props> {
-  renderLink = (route: any) => {
-    const { to, label, testId } = this.props;
-    return (
-      <li className={route.match ? "is-active" : ""}>
-        <Link to={to} {...createAttributesForTesting(testId)}>
-          {label}
-        </Link>
-      </li>
-    );
-  };
+const PrimaryNavigationLink: FC<Props> = ({ to, match, testId, label, className }) => {
+  const routeMatch = useRouteMatch({ path: match });
+  const history = useHistory();
 
-  render() {
-    const { to, match, activeOnlyWhenExact, testId } = this.props;
-    const path = match ? match : to;
-    return (
-      <Route
-        path={path}
-        exact={activeOnlyWhenExact}
-        children={this.renderLink}
-        {...createAttributesForTesting(testId)}
-      />
-    );
-  }
-}
+  return (
+    <a
+      onClick={() => history.push(to)}
+      className={classNames(className, "navbar-item", { "is-active": routeMatch })}
+      {...createAttributesForTesting(testId)}
+    >
+      {label}
+    </a>
```

Comment:
I think this could be a simple `Link` with a `to`, because `a` with a `history.push` on click, is the same as `<Link to={to}>{label}</Link>` and `Link` is rendered as an `a` with a `href`.